### DISCUSSION TITLE
can anyone fix my fail using REPLACE?

### DISCUSSION THREAD

**Discussion by iDINGUZ** [akosbalasko/yarle#622] [Q&A]
[
  {
    "type": "title",
    "regex": "[",
    "replace": "[["
  },
  {
    "type": "content",
    "regex": "[",
    "replace": "[["
  }
]

I use a single bracket for "WikiWords" in EVNERNOTE but want to see what [[backlinks]] look like in OBSIDIAN and the above didnt do it, I expect to run everything through twice to do the close "]" 
--AND hello/thanks!

**Comment by akosbalasko** [comment]
Hi @iDINGUZ ,
I think this should work, regex value has to have a syntactically correct regular expression, but as [ is a reserved character in regex (group separator), it has to be escaped:

``` markdown

[
{
"type": "title",
"regex": "\[",
"replace": "[["
},
{
"type": "content",
"regex": "\[",
"replace": "[["
},
{
"type": "title",
"regex": "\]",
"replace": "]]"
},
{
"type": "content",
"regex": "\]",
"replace": "]]"
}
]
```
Cheers,
akos

**Reply by iDINGUZ** [reply]
THANKS! escape characters are tricky I had seen double \\ and I think tried it
BUT
it seems my trouble is on the OBSIDIAN side or more expectation of it to build a backlink index I sent through some un yarl edited manually double bracketing in evernote and it didnt add up to LINKS my usecase is just journaling so I will probably learn an OUTLINER and just use CODE FOLDING instead of FOLDERS one big ugly .MD.

If you dont mind me asking where did your EVERNOTES end up in?

**Reply by akosbalasko** [reply]
Do you mean which note taking application I use currently? It is Obsidian, however I keep my eyes open to test any other brand new stuffs.
